addins/Cake.Handlebars.yml

```diff
@@ -5,6 +5,7 @@ Assemblies:
 Repository: https://github.com/agc93/Cake.Handlebars
 Author: Alistair Chapman
 Description: "A simple Cake addin for rendering and compiling Handlebars templates."
+Prerelease: "true"
```

Comment:
@Jericho Wonder if this is correct to switch to pre-release just because an newer pre-release exists?

Reply:
As mentioned previously, I was in favor of ignoring pre-release addins but @gep13 thought otherwise.